dissect/target/filesystems/tar.py

```diff
@@ -69,9 +70,9 @@ def detect(fh) -> bool:
         finally:
             fh.seek(offset)
 
-    def get(self, path) -> TarFilesystemEntry:
+    def get(self, path: str, relentry: FilesystemEntry = None) -> TarFilesystemEntry:
         """Returns a TarFilesystemEntry object corresponding to the given path."""
-        return self._fs.get(path)
+        return self._fs.get(path, relentry)
```

Comment:
```suggestion
        return self._fs.get(path, relentry=relentry)
```
Otherwise we may get in trouble if the function signature changes.